powershell/cmdlets/class.ts

```diff
@@ -1009,7 +1021,7 @@ export class CmdletClass extends Class {
                     // it's just a nested array
                     const p = getVirtualPropertyFromPropertyName(schema.language.csharp?.virtualProperties, valueProperty.serializedName);
                     if (p) {
-                      yield $this.WriteObjectWithViewControl(`(await response).${p.name}`, true);
+                      yield $this.WriteObjectWithViewControl(`${result.value}.${p.name}`, true);
```

Comment:
No sure if result.value is valid. result may be a primary type.

Reply:
result is a statement and result.value is string 'result'